gradle.properties

```diff
@@ -1,5 +1,5 @@
-ambariClientName=ambari-client20
-ambariClientVersion=2.0.27
+ambariClientName=ambari-client21
```

Comment:
Starting from now, we need to take care that ambari-client21 still remains compatible with 2.0.0